### DISCUSSION TITLE
The game doesn't automatically connect to a new audio device

### DISCUSSION THREAD

**Discussion by FoxyOwO199** [FakeFishGames/Barotrauma#14746] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

![image](https://github.com/user-attachments/assets/7bd6cf15-7181-4ced-89e5-9913570e9c54)

Please make it so the game will auto connect to any micrphone/headphones 
i have found it very frustrating that each time my cable moves a little and the headphones/microphone disconnects for even 3 seconds i need to restart the entire game just for it to recconnect. i can also just go to Audio & Voice chat and click on smth else then click back on what i was. but ye its very annoying. also i am sorry if i didnt do this correctly. this is like my second time using github lol

### Reproduction steps

1. open game
2. disconnect your headphones or microphone
3. replug and see how sounds dont work in game till you restart it.

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Other

### -

Happens in single player multiplayer and menu.

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/issues/7081

**Reply by FoxyOwO199** [reply]
damn thats from 2021 o,o its not duplicate just didnt see that

**Reply by Regalis11** [reply]
By a duplicate I mean it seems like the same issue of the audio device not getting updated. And yeah, we've been putting off addressing some of these audio issues for way too long - I think we've going to be finally moving the forwards in the next update. Sorry for the inconvenience!
